.travis.yml

```diff
@@ -47,10 +47,13 @@ install:
 
 before_script:
     - if [ "${TARGET}" != "${TARGET/e2e/}" ]; then
-        cd server &&
-        env SUPERDESK_URL='http://127.0.0.1:5000/api' SUPERDESK_CLIENT_URL='http://127.0.0.1:9000' SUPERDESK_TESTING='True' honcho start &
-        cd client &&
-        grunt server:travis --server='http://127.0.0.1:5000/api' --ws='ws://127.0.0.1:5100' &
+        cd server
+        && (env SUPERDESK_URL='http://127.0.0.1:5000/api' SUPERDESK_CLIENT_URL='http://127.0.0.1:9000' SUPERDESK_TESTING='True' honcho start &)
+        && cd ../client
+        && grunt build --server='http://127.0.0.1:5000/api' --ws='ws://127.0.0.1:5100'
+        && cd dist
+        && while [ ! -f index.html ]; do sleep .5; done
+        && (python3 -m http.server 9000 &)
```

Comment:
also u need to go back (ie `cd ..`) after, it's all executed as one script, so next command will be executed inside `client/dist`